### PR TITLE
Styling improvements

### PR DESCRIPTION
**Pull request by TicTak21**
Better spacing for Header, Greeting.
Width fix for keywords input.

### CODE REVIEW

File: src/components/search/SearchContainer.scss [hunk 1]
```diff
@@ -3,6 +3,10 @@
 .search-container {
   &-input {
     text-align: center;
+
+    &-field {
+      width: 70%;
```
Comment (trybick) [inline src/components/search/SearchContainer.scss]: I think 70% might too big - could we go with 40%?

**Review by trybick — COMMENTED** [review]
The padding and margin adjustments look good, thank you 👍 👏 
One comment about the input.

Two other things I realized I would like to add. If you want to add them that's great or I can add them to another issue, either way:

- Change box shadow on 'greeting-card' class to `-1px 2px 14px rgba(0,0,0,.25), 1px 1px 3px rgba(0,0,0,.1)`
- Change search input placeholder to "Enter keywords (optional)"

File: src/components/search/SearchContainer.scss (hunk 1 above)
Comment (TicTak21) [reply src/components/search/SearchContainer.scss]: Sure. I've already made changes.

**Comment by trybick** [comment]
@TicTak21 Looks great, thank you!

**Pull request merged**